docs/tutorials/data_visualization.ipynb

```diff
@@ -55,8 +55,8 @@
     "\n",
     "%matplotlib inline\n",
     "\n",
-    "from IPython.core.display import display, HTML\n",
-    "display(HTML(\"<style>.container { width:98% !important; }</style>\"))"
+    #"from IPython.core.display import display, HTML\n",
```

Comment:
json has no comments, just delete the lines (or move the comment into the string

Reply:
just saw... :-C

Reply:
You can also just remove the "core"